numpy/testing/_private/utils.py

```diff
@@ -929,10 +937,37 @@ def assert_array_equal(x, y, err_msg='', verbose=True):
     >>> x = np.full((2, 5), fill_value=3)
     >>> np.testing.assert_array_equal(x, 3)
 
+    Use `strict` to raise an assertion when comparing a scalar with an array:
```

Comment:
Either "raise an exception" or "raise an AssertionError" (not "raise an assertion").